### ISSUE TITLE
v-bind:a.sync is reported

### ISSUE THREAD

**Issue by Akryum** [vuejs/eslint-plugin-vue#27]
```html
<ui-period-picker :start.sync="startDate" :end.sync="endDate" />
```

# Expected

No error.

# Happening

Produces the `'v-bind' directives don't support the modifier 'sync'` error.

**Comment by michalsnik** [comment]
Released as `v3.1.0`, still tagged as beta though. Thanks for catching this @Akryum :)
